src/Selenium2Library/utils/librarylistener.py

```diff
@@ -0,0 +1,18 @@
+import events as event
+from robot.api import logger
+
+class LibraryListener(object):
+
+	ROBOT_LISTENER_API_VERSION = 2
+
+	def _start_suite(self, name, attrs):
```

Comment:
It would be OK not to have the leading underscore because this class is not used as a library.